integration_test/third_party_apps_data/applications/aerospike/sles/install

```diff
@@ -3,19 +3,33 @@ set -e
 # aerospike-init requires `python` in $PATH
 sudo ln -s /usr/bin/python3 /usr/bin/python
 
-curl -L -o aerospike.tar.gz https://github.com/aerospike/aerospike-server/releases/download/5.7.0.23/aerospike-server-community-5.7.0.23.tar.gz
+if [[ "$(uname -m)" == aarch64 ]]; then
+    # 6.2 is the first version with ARM64 support
+    curl -L -o aerospike.tar.gz https://github.com/aerospike/aerospike-server/releases/download/6.2.0.8/aerospike-server-community-6.2.0.8.tar.gz
+else
+    curl -L -o aerospike.tar.gz https://github.com/aerospike/aerospike-server/releases/download/5.7.0.23/aerospike-server-community-5.7.0.23.tar.gz
+fi
 tar -xzf aerospike.tar.gz
 cd aerospike-server
 ./bin/aerospike init
 
 # aerospike-init creates an invalid config
 sed -i 's|default-ttl.*||g' etc/aerospike.conf
+if [[ "$(uname -m)" == aarch64 ]]; then
```

Comment:
what do you think, does this need its own comment or is `# aerospike-init creates an invalid config` sufficient for this case too?

Reply:
@martijnvans I think it's still a sufficient comment (it just creates "more" of an invalid config for arm64 😄 ), but let me know if you'd like an additional comment.